helm/oauth2-proxy/templates/ingress.yaml

```diff
@@ -5,7 +5,7 @@
 {{- $ingressPathType := .Values.ingress.pathType -}}
 {{- $extraPaths := .Values.ingress.extraPaths -}}
 {{- $apiV1 := false -}}
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" -}}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress" -}}
 apiVersion: networking.k8s.io/v1
 {{- $apiV1 = true -}}
 {{- else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" -}}
```

Comment:
Should `/Ingress` be specified also here?

Reply:
I'd prefer to keep this PR as complete as possible. 😜 
For me, it makes sense to check inside the Ingress manifest if ".Capabilities.APIVersions.Has" for the Ingress specific API in the whole file, not only in one point.

Reply:
Changed it.